test/unit/ajax.js

```diff
@@ -1590,6 +1592,19 @@ module( "ajax", {
 				ok( false, "error" );
 			}
 		} );
+
+		if ( jQuery.support.cors ) {
+			ajaxTest( "gh-1684 - jQuery.ajax() - support cross-domain PATCH in IE10-11", 1, {
+				url: "http://httpbin.org/patch",
```

Comment:
>> I don't understand. Spartan doesn't have document.documentMode and doesn't have window.ActiveXObject. It will be treated as every non-IE browser so I don't see a problem here.
>
> It should be like that, but until Spartan is officially released we can't know that, hence me noting this as a "likely scenario" not a definite fact, but that is beside the point, purely hypothetical - an example.

Well, until Spartan is released theoretically anything may happen so we cannot test against it. I can swap the `documentMode` check here for the `window.ActiveXObject` one, this would be closer to what we're doing here as the whole logic is skipped in browsers without `ActiveXObject`.

> Since these tests are so specific, they can't be preventive, only regressive.

That's still important. Preventing testing here would be adding more cross-domain request types than just PATCH.